### PR TITLE
Add new package for r-spdl

### PR DESCRIPTION
**Pull request by Shelnutt2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Shelnutt2** [comment]
@conda-forge/help-r this package PR is ready for review.

**Review by jdblischak — APPROVED** [review]

File: recipes/r-spdl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    from the 'fmtlib' library (that is also 'std::format()' in 'C++20' or later).
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - shelnutt2
+    - eddelbuettel
```
Comment (dbast) [inline recipes/r-spdl/meta.yaml:55]: @shelnutt2 the recipe looks good. Adding Dirk Eddelbuettel here requires his consent regarding becoming conda recipe maintainer of the package.

@eddelbuettel Are you fine with / interested in being conda-forge recipe maintainer of your cran spdl package?

**Comment by eddelbuettel** [comment]
@dbast To be honest I would rather not as _I_ do not use Conda at all.  And I have enough open-source balls in the air with the packages I look after for Debian (this one one include), at CRAN, via Rocker, now r2u, ...  

We have other simple packages of mine already in Conda and they work just fine without my input.  My colleagues took great care of this, and I have in the past helped where needed (and sometimes was able to assist with a bit of CRAN experience). I'd be fine keeping that as is.

File: recipes/r-spdl/meta.yaml (hunk 1 above)
Comment (jdblischak) [reply recipes/r-spdl/meta.yaml:55]: @Shelnutt2 Can you please remove Dirk from the list of recipe co-maintainers? You can also add me if you like

Comment (eddelbuettel) [reply recipes/r-spdl/meta.yaml:55]: Sorry I first the 'formal consent' part and potential blockers. 

I am happy to be one of several maintainers for this.

**Comment by cbrueffer** [comment]
Thanks everyone!

**Pull request merged**